staff/carlos-calvo/mailchimpnew/index.html

```diff
@@ -0,0 +1,131 @@
+                </div>
+            </nav>  
+        </header>
+        <section class="subuppersection">
+            <div>
+                <img src="./images/easy-email-newsletters-banner-png8.png" alt="">
+            </div>
+            <div class="subuppersection__rightside">
+                <h1>Easy Email Newsletters</h1>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+                        aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                        Duis aute irure dolor in reprehenderit in voluptate v</p>
+            <button class="botobasic botobasic--red">Sign Up Free</button>
+            </div>
+        </section>  
+    </section>
+    <!--Hasta aqui zona azul-->
+    <!--A partir de aqui zona gris y 4 columnas-->
+    <main><!--zona gris-->
+        <div class="cuadro"><!--cuadro principal-->
+            <div class="cuadro__Whatsnew">
+                <h2>What's new</h2>
+            </div>
+            <div class="cuadro__fourcolcontainer">
+                <div class="eachcolumn">
+                    <img src="./images/templates.jpg">
+                    <h5>New Feature</h5>
+                    <h3>More Templates</h3>
+                    <p>We jsut added a botadload of beatufil templates to mailchimp and they're free for everyone</p>
+                    <button class="botobasic boto--little">Learn More</button>
+                </div>
+                <div class="eachcolumn">
+                    <img src="./images/denim2.jpg">
+                    <h5>New Feature</h5>
+                    <h3>More Templates</h3>
+                    <p>We jsut added a botadload of beatufil templates to mailchimp and they're free for everyone</p>
+                    <button class="botobasic boto--little">Learn More</button>
+                </div>
+                <div class="eachcolumn">
+                    <img src="./images/unfurlr-thumb.jpg">
+                    <h5>New Feature</h5>
+                    <h3>More Templates</h3>
+                    <p>We jsut added a botadload of beatufil templates to mailchimp and they're free for everyone</p>
+                    <button class="botobasic boto--little">Learn More</button>
+                </div>
+                <div class="eachcolumn">
+                    <img src="./images/golden-monkeys.jpg">
+                    <h5>New Feature</h5>
+                    <h3>More Templates</h3>
+                    <p>We jsut added a botadload of beatufil templates to mailchimp and they're free for everyone</p>
+                    <button class="botobasic boto--little">Learn More</button>
+                </div>
+            </div>
+        </div>
+    </main>
+    <footer>
+        <div class="fourcolumns"> <!--Div para las columnas-->
+            <div class="fourcolumns__footercolumn"> <!--Cada columna-->
```

Comment:
Maybe use nav -> ul -> li -> a for all 4 columns?

Reply:
It makes sense. Changed in next version.